scripts/clean.php

```diff
@@ -44,6 +44,9 @@ function getHTTPInfo($url, $followLocation = false)
     $total_to_delete++;
     echo "(" . $total_to_delete . ") deleted: " . $row['longURL'] . PHP_EOL;
   }
+  //Impletement deletion of any short URLS which have not been visited in two years
+  $mysqli->query("DELETE FROM tblURLs WHERE DATE_ADD(lastRedirectDate, INTERVAL 2 YEAR) < CURRENT_TIMESTAMP()");
```

Comment:
Should exclude NULL rows

Reply:
I ran the clean script with a NULL value in the "lastRedirectDate" column and it was not deleted. Does this mean that the script is already excluding NULL values?

Reply:
Sorry for the confusion, I was sure how the check DATE_ADD(lastRedirectDate, INTERVAL 2 YEAR) < CURRENT_TIMESTAMP()` would evaluate for NULL value of `lastRedirectDate`.  If it did result in a delete you could have the delete statement to: `DELETE FROM tblURLs WHERE lastRedirectDate IS NOT NULL AND (lastRedirectDate, INTERVAL 2 YEAR) < CURRENT_TIMESTAMP()`